python/lsst/ts/standardscripts/maintel/m1m3/raise_m1m3.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["RaiseM1M3"]
+
+import time
+
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+from lsst.ts import salobj
+
+
+class RaiseM1M3(salobj.BaseScript):
+    """Raise M1M3 mirror.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Raise M1M3")
```

Comment:
I believe one of the reasons that your unit tests keep running forever is due to the "usage."  
@tribeiro is this one of the cases where you have to define the usage as `DryTest`? 
Here is an example: [track_target_and_take_image_gencam.py#L48
](https://github.com/lsst-ts/ts_standardscripts/blob/48f2da79c0ea2e6f765407d0b6cccef8f1facc12/python/lsst/ts/standardscripts/maintel/track_target_and_take_image_gencam.py#L48)

Reply:
👍 I have added the usage to
`mtcs_usage = None if add_remotes else MTCSUsages.DryTest` 
The test_run runs forever 👎

Reply:
Yeah... I don't know a solution to this. It happened to me too, and the usage was the solution. @tribeiro, any other suggestion?